form/create_form_type_extension.rst

```diff
@@ -131,12 +131,16 @@ The ``extended_type`` key of the tag is the type of field that this extension sh
 be applied to. In your case, as you want to extend the ``Symfony\Component\Form\Extension\Core\Type\FileType``
 field type, you will use that as the ``extended_type``.
 
+.. versionadded:: 3.3
+    Prior to Symfony 3.3, you needed to define form extension services as ``public``.
```

Comment:
"[...] define type extension services [...]"

This is needed because there are also form extensions, but we do not register them.